### DISCUSSION TITLE
MAUI native layout/controls and their native counterparts

### DISCUSSION THREAD

**Discussion by alphac05** [dotnet/maui#17393] [Q&A]
For Xamarin Forms, there is a useful documentation page that gives the list of view, controls... and their counterparts for each native platform:

https://learn.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/custom-renderer/renderers

It is useful to know, for example, that a `Frame` translates into a `ViewGroup` in Android:

Layout | Renderer | iOS | Android | Android (AppCompat) | UWP
-- | -- | -- | -- | -- | --
...
Frame | FrameRenderer | UIView | ViewGroup | CardView | Border
...

Is there anything similar for MAUI?

**Comment by xtuzy** [comment]
I only find https://learn.microsoft.com/en-us/dotnet/maui/user-interface/handlers/ , you can search Handler in this Repository , CreatePlatformView() will return native view.
